### DISCUSSION TITLE
Welcome to IEA-22-280-RWT Discussions!

### DISCUSSION THREAD

**Discussion by fzahle** [IEAWindSystems/IEA-22-280-RWT#88] [Announcements]
## 👋 Welcome to IEA 22 MW RWT discussions!
  We’re using Discussions as a place to connect users of the IEA 22 MW RWT. We hope that you:
  * Ask questions you’re wondering about.
  * Share news about projects in which you use the IEA 22 MW RWT
  * Share work you're doing or publications using the IEA 22 MW RWT.
  * Engage with other community members.

Please continue to use the issues tracker to report problems and bugs with the IEA 22 MW RWT.

**Comment by NicholasMRobinson** [comment]
Hello,

It looks like lots of great work has been done here. I am looking for much more pedestrian information such as power curves and thrust coefficient curves by wind speed. Have those been published anywhere please? I have the PDF and Excel files which contain some of the information I am looking for. I realise I could generate my own power curve from the text in the PDF but if someone has already got these tables (especially for Ct) and is willing to share, that would be great.

I make software (Openwind) that is used by the collegiate wind competition and so am trying to update my list of publicly available turbines.

Thanks,

Nick Robinson

**Comment by gbarter** [comment]
Hi Nick,

Please see the "Rotor Performance" tab in the Excel file in [this folder](https://github.com/IEAWindSystems/IEA-22-280-RWT/tree/main/Documentation).  Rotor power and thrust are listed as a function of wind speed.  You would have to compute the Cp/Ct coefficients from there.

Cheers,
Garrett

**Comment by NicholasMRobinson** [comment]
Just came back here to say I found it. Thanks :)

There are columns for power in MW and thrust coefficient so that's great. Am I right in thinking this is effectively a zero TI power curve as opposed to a 10 minute averaged power curve with say 10% TI? I can use the power curve working group method to adjust for 10 minute averaging if so.

Thanks again,

Nick

**Comment by gbarter** [comment]
The performance data was generated by HAWC2.  I'll defer to @fzahle or @tkbadtu as to how idealized the power curve numbers are.

**Comment by ptrbortolotti** [comment]
Hi, you can also find power and thrust values here https://github.com/IEAWindSystems/IEA-22-280-RWT/tree/main/outputs, for steady state and turbulent inflow. These are the values documented in this publication https://iopscience.iop.org/article/10.1088/1742-6596/2767/5/052042/meta

**Comment by ZIM797** [comment]
I have created the submersible geometry of the platform. Now, how do I create WAMIT files? Can anyone help me, please? I don't know which software to use for this or how to run it.
